src/ripple/app/paths/impl/StrandFlow.h

```diff
@@ -348,24 +348,26 @@ class ActiveStrands
    @param strands Each strand contains the steps of accounts to ripple through
                   and offer books to use
    @param outReq Amount of output requested from the strand
-   @param flowParams Constraints and options on the payment
-   @param logs Logs to write journal messages to
+   @param partialPayment If true allow less than the full payment
+   @param offerCrossing If true offer crossing, not handling a standard payment
+   @param limitQuality If present, the minimum quality for any offer taken
```

Comment:
This quality is for a given strand, but the comment says is for offers only: s/offer/strand